package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "beccaccino",
-  "version": "2.0.0",
+  "version": "2.0.0-upgrade-dephs.0",
```

Comment:
Maybe it's a typo? You mean `deps`?

Reply:
Hi @CDimonaco! Yes, thank you, fortunately, I haven't published yet :)